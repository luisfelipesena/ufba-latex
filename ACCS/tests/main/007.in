3486 26186118 39366826
