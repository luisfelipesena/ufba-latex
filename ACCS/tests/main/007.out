1030864350924954
